src/main/java/com/example/myInventory/controllers/StoreController.java

```diff
@@ -0,0 +1,81 @@
+package com.example.myInventory.controllers;
+
+import com.example.myInventory.models.Inventory;
+import com.example.myInventory.models.Store;
+import com.example.myInventory.models.data.InventoryDao;
+import com.example.myInventory.models.data.StoreDao;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import javax.validation.Valid;
+
+@Controller
+@RequestMapping("store")
+public class StoreController {
+
+    @Autowired
+    private StoreDao storeDao;
+
+    @Autowired
+    private InventoryDao inventoryDao;
+
+    @RequestMapping(value = "")
+    public String index(Model model){
+
+        model.addAttribute("title","Username Stores");
+        model.addAttribute("stores", storeDao.findAll());
+
+        return "store/index";
+    }
+
+    @RequestMapping(value = "add", method = RequestMethod.GET)
+    public String addStoreForm(Model model){
+
+        model.addAttribute("title","Add a Store");
+        model.addAttribute(new Store());
+
+        return "store/add";
+    }
+
+    @RequestMapping(value = "add", method = RequestMethod.POST)
+    public String processAddStoreForm(@ModelAttribute @Valid Store store, Errors errors, Model model,
+                                      @RequestParam String inventoryName){
```

Comment:
Not super important right now, but look into `@RequestBody` it basically maps the body of a request to an object. that way you dont have to instantiate a new `Inventory inventory` every time.

it would look something like 

```
@RequestBody Inventory inventory
```
if inside the Inventory object there was a field called `inventoryName`

Just a thought!